src/Raven.Studio/typescript/viewmodels/database/settings/databaseSettings.ts

```diff
@@ -272,6 +284,21 @@ class databaseSettings extends viewModelBase {
                 ]
             }
         });
+
+        this.columnPreview.install(".summary-list-container", ".js-summary-details-tooltip",
+            (details: models.settingsEntry,
+             column: textColumn<models.settingsEntry>,
+             e: JQueryEventObject, 
+             onValue: (context: any, valueToCopy?: string) => void) => {
+                if (column.header === "Origin") {
```

Comment:
can we invert if to avoid empty code branch?

Reply:
done